NuGet.config

```diff
@@ -6,5 +6,6 @@
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
     <add key="Orchard2" value="https://www.myget.org/F/orchard2/api/v3/index.json" />
     <add key="icunet" value="https://www.myget.org/F/icu-dotnet/api/v3/index.json" />
+    <add key="Orchard.Modules" value="https://www.myget.org/F/orchard-modules/api/v3/index.json" />
```

Comment:
Yes, you need it for testing because the app references a demo module as a package consumed from this feed. At the end, i'll remove this package reference and also this line in nuget config.